test/TestAsset/Anonymous/WithAnonymousClass.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
```

Comment:
Three changes:

- Reference the zend-file repository, using https.
- Since this file is just being introduced, change the copyright date to simply `2017`.
- Point to https://github.com/zendframework/zend-file/blob/master/LICENSE.md for the license.